math/cmath/include/algebra/math/impl/cmath_generator.hpp

```diff
@@ -0,0 +1,49 @@
+/** Algebra plugins library, part of the ACTS project
+ *
+ * (c) 2022 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+// Project include(s).
+#include "algebra/qualifiers.hpp"
+
+// System include(s).
+#include <cassert>
+#include <cstddef>
+#include <type_traits>
```

Comment:
I don't think you use anything out of any of these headers. :thinking:

But you do use [std::min](https://en.cppreference.com/w/cpp/algorithm/min). For which you need to include [\<algorithm\>](https://en.cppreference.com/w/cpp/header/algorithm) instead.